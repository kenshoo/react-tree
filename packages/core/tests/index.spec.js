import React from "react";
import { shallow } from "enzyme";
import HelloWorld from "../src";
import toJson from "enzyme-to-json";

describe("core component", () => {
  test("will render", () => {
    const cmp = shallow(<HelloWorld />);
    expect(toJson(cmp)).toMatchSnapshot();
  });
});
