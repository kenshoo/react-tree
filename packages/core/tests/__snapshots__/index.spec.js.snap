// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`core component will render 1`] = `
<div>
  Hello World
</div>
`;
